recipients_app/lib/core/cubits/payment/payments_cubit.dart

```diff
@@ -94,4 +98,123 @@ class PaymentsCubit extends Cubit<PaymentsState> {
       );
     }
   }
+
+  Future<PaymentsUiState> _mapPaymentsUiState() async {
```

Comment:
This method should be private because it should never be accessed from outside the cubit, right?

Reply:
yeah... I always forget about the `_` 🥶